core/sql/exp/ExpLOBprocess.cpp

```diff
@@ -575,7 +575,7 @@ Lng32 main(Lng32 argc, char *argv[])
     // setup log4cxx
     QRLogger::initLog4cxx(QRLogger::QRL_LOB);
     // initialize lob globals
-    lobGlobals = new ExLobGlobals();
+    lobGlobals = new ExLobGlobals(NULL);
```

Comment:
Thanks for avoiding the default value for the param. But, I think sending heap as NULL might make this program to fail. Have you tested this program? If you have tested this program already, then this PR is ready to be merged.

Reply:
The ExpLOBprocess.cpp is dormant code for now. If we enable it and create a LOB process to offload some operations like GC etc, we will need to create and setup globals , send messages form the master process to it etc .  For now I just had to ensure it compiled.